follower_default_send_mail_false/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,2 @@
+This module Sets the send_mail field to False by default when adding a new follower
+and the partners will not receive an email warning they have been added in the document's followers.
```

Comment:
```suggestion
This module sets send_mail to False by default when adding a new follower
to avoid unnecessary email notification of them being added as a follower.
```